modules/ob-creating-a-buildpacks-build.adoc

```diff
@@ -0,0 +1,264 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ob-creating-a-buildpacks-build_{context}"]
+= Creating a buildpacks build
+
+Learn how to create a buildpacks build and push the created image into the target registry.
+There are two buildpacks `ClusterBuildStrategy` options are available for {builds-title}:
+
+* `buildpacks` strategy
+* `buildpacks-extender` strategy
+
+[NOTE]
+====
+The `buildpacks-extender` strategy is compatible with the experimental buildpacks `extender` lifecycle phase. You can use this strategy e.g for builds based on Node.js. However, the `extender` phase does not support Quarkus or any other Java-based builds. For such builds, you should use the `buildpacks` strategy instead.
+====
+
+[IMPORTANT]
+====
+Buildpacks is a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have installed the `oc` CLI.
+* You have created the project where your final application image will be stored by using the command `oc new-project buildpacks-example`.
+* Optional: You have installed the `shp` CLI.
+
+
+[IMPORTANT]
+====
+Using `shp` CLI with buildpacks requires additional permissions setup
```

Comment:
```suggestion
Using `shp` CLI with buildpacks requires additional permissions setup that
```

Reply:
It looks like you have a hard return between line 33 and 34. Please correct this.